### PR TITLE
Documented some documentation achievements

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

File: achievements/documentation.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+This resulted in a diverse set of work done. Below is a short summary
+of all work done, but please have a look at [all GitHub issues and pull
+requests](https://github.com/symfony/symfony-docs/pulls?q=label%3A"⭐️+EU-FOSSA+Hackathon")
+handled during the weekend.
+
+### Documenting Features that were Missing for Years
+
+While Symfony puts lots of effort into documenting all new features,
+it cannot be avoided that sometimes great features as missed. During the
```
Comment (OskarStark) [inline achievements/documentation.md]: is "as" correct here? 🤔

**Review by OskarStark — COMMENTED** [review]
Super article Wouter 👏

File: achievements/documentation.md (hunk 1 above)
Comment (wouterj) [reply achievements/documentation.md]: Thanks, fixed!

File: achievements/documentation.md [hunk 2]
```diff
@@ -1 +1,64 @@
+# Documentation
 
+The EU-FOSSA hackathon allowed us to, for the first time ever, gather
+4 of the doc team members together to work on the documentation. Also,
+by combining the doc team with all code contributors, we managed to
+help code contributors to write doc PRs for their contributions. Being
+able to directly help people understand the reStructuredText syntax and
+logics of writing docs was a huge help to get more features documented.
+
+This resulted in a diverse set of work done. Below is a short summary
```
Comment (jfhovinne) [inline achievements/documentation.md]: 'a diverse set of achievements' maybe ? So we don't repeat 'work done'.

File: achievements/documentation.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+of all work done, but please have a look at [all GitHub issues and pull
+requests](https://github.com/symfony/symfony-docs/pulls?q=label%3A"⭐️+EU-FOSSA+Hackathon")
+handled during the weekend.
+
+### Documenting Features that were Missing for Years
```
Comment (jfhovinne) [inline achievements/documentation.md]: should be level 2

File: achievements/documentation.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+Among this, [@Toflar](https://github.com/Toflar) worked on [documenting the
+changes in `.env` files in Symfony 4.2](https://symfony.com/doc/current/components/dotenv#usage)
+While the recommendations changed (from `.env.dist` to `.env.local`), the
+docs were never updated accordingly.
+
+[@Nyholm](https://github.com/Nyholm) combined the knowledge of almost everyone
+around to write a brand new [Cache guide](https://symfony.com/doc/current/cache.html).
+As the component has changed quite often and involved many code contributors, we're
+very happy there was finally an occasion where the documentors and code contributors
+couuld combine their knowledge about this topic. This will be a great help for all futur
```
Comment (jfhovinne) [inline achievements/documentation.md]: 'could' and 'future'

File: achievements/documentation.md [hunk 5]
```diff
@@ -1 +1,64 @@
...
+
+### Finishing old Pull Requests
+
+The documentation has around 100 open PRs, most of which are open for months or even
+years. During the hackathon, developers were encouraged to finish these old PRs.
+[@llaakkkk](https://github.com/llaakkkk) worked on finishing a [new article on Message
+Transports](https://github.com/symfony/symfony-docs/pull/11331), [@Toflar](https://github.com/Toflar)
+finished [adding Doctrine ORM installation instructions](https://github.com/symfony/symfony-docs/pull/11332),
+and some more old PRs were closed and fixed by doc team members with the help of
+[@dbrumann](https://github.com/dbrumann), who focussed for the complete hackathon on
```
Comment (jfhovinne) [inline achievements/documentation.md]: 'during' instead of 'for' ?

File: achievements/documentation.md [hunk 6]
```diff
@@ -1 +1,64 @@
...
+the docs.
+
+### Documenting new Features
+
+At the second day, the docs corner became quite busy, as all developers that created
```
Comment (jfhovinne) [inline achievements/documentation.md]: 'During' instead of 'At' ?
all developers who created

File: achievements/documentation.md [hunk 7]
```diff
@@ -1 +1,64 @@
...
+new features on the first day wanted to directly document it. "A feature does not exists
```
Comment (jfhovinne) [inline achievements/documentation.md]: 'document them'
'does not exist'

File: achievements/documentation.md [hunk 8]
```diff
@@ -1 +1,64 @@
...
+without documentation" became reality this weekend! In total, 39 different authors created
+61 pull requests during the weekend!
+
+### The Hackathon ended, the Contributions did not!
+
+The hackathon gave all participants lots of positive energy, something that became obvious
+during the days after the hackathon. The stream of new PRs continued to be impressively high,
+with over 24 new pull requests created at Monday (that almost equals the achievements of the
```
Comment (jfhovinne) [inline achievements/documentation.md]: on Monday

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Very nice, thanks. Minor fixes needed.

**Comment by wouterj** [comment]
Thanks for the review @jfhovinne! I've fixed all your comments

**Review by jfhovinne — APPROVED** [review]

**Comment by jfhovinne** [comment]
Thanks!

**Pull request merged**